docs/upgrade/3.4.4.rst

```diff
@@ -1,4 +1,4 @@
-.. _upgrade-to-3.4.4:
+we have.. _upgrade-to-3.4.4:
```

Comment:
Is this correct to add "we have" here? Looks odd to me.

Reply:
Thank you, Mark. This is an error on my part. I erased that. I will be more careful.